### DISCUSSION TITLE
eBook TTS

### DISCUSSION THREAD

**Discussion by adambland01** [advplyr/audiobookshelf-app#1246] [Ideas]
ABS app is a great replacement for all of my other audiobook apps.  I know the eBook features are probably lower priority, but the ability to have TTS for eBooks would be a great way to provide a unified audiobook experience for all mixed libraries within a single app.

Love the clean simple UI!

**Comment by nichwall** [comment]
https://github.com/advplyr/audiobookshelf/issues/601
